src/test/java/homeworks/POM/Pages/LoginPageHmw.java

```diff
@@ -0,0 +1,39 @@
+package homeworks.POM.Pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class LoginPageHmw extends BasePageHmw{
+By emailFieldLocator = By.cssSelector("[type='email']");
+By passwordFieldLocator = By.cssSelector("[type='password']");
+By submitButtonLocator = By.cssSelector("[type='submit']");
```

Comment:
**I recommend:**  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

```
  private By emailFieldLocator = By.cssSelector("[type='email']");
  private By passwordFieldLocator = By.cssSelector("[type='password']");
  private By submitButtonLocator = By.cssSelector("[type='submit']");
```